packages/framework/src/Model/Statistics/StatisticsRepository.php

```diff
@@ -83,4 +86,76 @@ function (array $item) {
             $query->getResult()
         );
     }
+
+    /**
+     * @param \DateTimeImmutable $startDateTime
+     * @param \DateTimeImmutable $endDateTime
+     * @return int
+     */
+    public function getNewCustomersCountBetweenDates(DateTimeImmutable $startDateTime, DateTimeImmutable $endDateTime)
+    {
+        $resultSetMapping = new ResultSetMapping();
```

Comment:
don't you thinks that many lines are identical?

Reply:
Well they are similar, but no identical... I think this is not DRY violation.